packages/ia-activity-indicator/demo/index.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
```

Comment:
This file's got a number of easy eslint errors that can be resolved, like use of the `var` statement and passing function refs before they're defined (that's a bit of a strange one).

Reply:
This is a demo file. It's just demoing functionality, not really shipping code.

Reply:
It gets run through eslint with the `yarn lint` script, so if we're not going to fix the errors it should be added to the .eslintignore file.

Reply:
It does? Interesting.. I wonder why that didn't fail the build 🤔

Reply:
Yeah, I wondered the same thing. Maybe you have a global ignore?

Reply:
No, we just weren't linting during the `test` script so it was passing. I added a lint step to the test script.